app/src/main/java/ru/dimagor555/passwordmanager/MainActivity.kt

```diff
@@ -23,6 +23,10 @@ class MainActivity : ComponentActivity() {
         setContent {
             PasswordManagerTheme {
                 val navController = rememberNavController()
+                /*
+                 FIXME: 04.12.2021 I'm strongly recommend to use constant or enum as you alraedy
+                        did with password length, for example
+                 */
```

Comment:
Yes, I totally agree. I will create navigation module in the future, but now this code is a glue for launching the application. At the moment I'm focused on features

Reply:
please, don't judge me strictly and don't follow my recommendations blindly. I've just tried to cover as much topics as I could